src/ViewModels/src/DeploymentDialog/IDeploymentDialogViewModel.cs

```diff
@@ -4,8 +4,9 @@ namespace Tanzu.Toolkit.ViewModels
 {
     public interface IDeploymentDialogViewModel
     {
-        string ProjectDirPath { get; }
+        string PathToProjectRootDir { get; }
         string ManifestPath { get; set; }
+        string SelectedDeploymentDirectoryPath { get; set; }
```

Comment:
Is the `Selected` prefix needed?  Seems superfluous.

Reply:
Nope! I think it may have been an effort to distinguish it from other path-related properties (e.g. this is the one the user _selected_, not the directory containing the app's source files), but looking at those other props now I think `DeploymentDirectoryPath` is plenty descriptive. Changing now